src/status_im2/contexts/onboarding/common/intro/style.cljs

```diff
@@ -0,0 +1,62 @@
+(ns status-im2.contexts.onboarding.common.intro.style
+  (:require
+    [react-native.platform :as platform]
+    [quo2.foundations.colors :as colors]))
+
+(def progress-bar-container
+  {:background-color colors/neutral-100
+   :flex-direction   :row
+   :margin-vertical  16})
+
+(defn progress-bar-item
+  [index position end?]
+  {:height       2
+   :flex         1
+   :border-width 1
+   :border-color (if (= index position) colors/white colors/white-opa-10)
+   :margin-right (if end? 0 8)})
+
+(def carousel
+  {:height             92
+   ;; (padding-top) This insets issue needs a consistent implementation across all screens.
+   :padding-top        (if platform/android? 0 44)
```

Comment:
why is it different on android ?

Reply:
or should I make use of the insets here? I believe for the redesign it is consistently like this however since we are hiding the top bar. Perhaps we can just put this in as the default inset for navigation screens and then clean up all the screens in a refactor? @Parveshdhull @flexsurfer wdyt?

Reply:
yes i see a lot of this in the new code, and it looks like a misconception, we shouldn't do it in every place

Reply:
okay, how would you feel if I leave it as is in this pr and do a follow up to configure all the screens setting this value in one place, i.e navigation screens?

Reply:
yes sure it should be done in separate PR